tests/foreman/api/test_role.py

```diff
@@ -1631,16 +1705,49 @@ def test_positive_assign_org_admin_to_ldap_user_group(self):
 
         :CaseLevel: System
 
-        :CaseAutomation: NotAutomated
+        :CaseAutomation: Automated
         """
+        group_name = gen_string("alpha")
+        password = gen_string("alpha")
+        org_admin = self.create_org_admin_role(
+            orgs=[create_ldap['authsource'].organization[0].id],
+            locs=[create_ldap['authsource'].location[0].id],
+        )
+        # Creating Domain resource in same taxonomies as Org Admin role to access later
+        domain = self.create_domain(
+            orgs=[create_ldap['authsource'].organization[0].id],
+            locs=[create_ldap['authsource'].location[0].id],
+        )
+        users = [
+            entities.User(
+                login=gen_string("alpha"),
+                password=password,
+                organization=create_ldap['authsource'].organization,
+                location=create_ldap['authsource'].location,
+            ).create()
+            for _ in range(2)
+        ]
+        user_group = entities.UserGroup(name=group_name, user=users, role=[org_admin]).create()
+        # Adding LDAP authsource to the usergroup
+        entities.ExternalUserGroup(
+            name='foobargroup', usergroup=user_group, auth_source=create_ldap['authsource']
+        ).create()
+
+        for user in users:
+            sc = ServerConfig(
+                auth=(user.login, password),
+                url=create_ldap['sat_url'],
+                verify=False,
+            )
+            # Accessing the Domain resource
+            entities.Domain(sc, id=domain.id).read()
```

Comment:
No wrapping into `with pytest.raises(HTTPError):` here?

Reply:
Not required here as this is a positive test scenario and the domain resource we are accessing here is in the same taxonomy as the org_admin_role, hence the user should be able to access the Domain resource created.